packages/api-designer/src/generators/mock-server-generator.ts

```
@@ -0,0 +1,144 @@
+ * Generate mock data from JSON Schema
+ */
+function generateMockFromSchema(schema: any): any {
+  if (!schema) return null;
+
+  if (schema.$ref) {
+    return { id: 1, name: 'Mock Data' };
+  }
+
+  switch (schema.type) {
+    case 'object':
+      const obj: any = {};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)